components/ArticleCard/ArticleCardFooter.tsx

```diff
@@ -0,0 +1,76 @@
+import classNames from 'classnames';
+
+import { dotDate } from '@/utils/dateConversion';
+import { CardFooter } from '@/components/ui/card';
+import WriterProfile from '@/components/WriterProfile';
+import LikeCount from '@/components/LikeCount';
+import useUserStore from '@/stores/useUser.store';
+import KebabDropDown from '@/components/KebabDropDown';
+
+/**
+ * @param {boolean} props.isBestCard - 게시글 데이터
+ * @param {object} props.writer - 게시글 작성자 id, nickname
+ * @param {string} props.createdAt - 게시글 생성일
+ * @param {number} props.likeCount - 게시글 좋아요 수
+ * @returns {JSX.Element} 게시글 카드의 프로필, 생성일, 좋아요가 포함된 Card Footer 컴포넌트
+ */
+function ArticleCardFooter({
+  isBestCard,
+  writer,
+  likeCount,
+  createdAt,
+}: {
+  isBestCard: boolean;
+  writer: { nickname: string; id: number };
+  createdAt: string;
+  likeCount: number;
+}) {
+  const { user: userData } = useUserStore();
+
+  const cardFooterStyled = classNames(
+    'p-0 mo:items-center',
+    !isBestCard
+      ? 'mt-pr-24 flex justify-between mo:mt-pr-25'
+      : 'mt-auto flex-col',
+  );
+  const createdAtStyled =
+    "relative pl-pr-32 before:absolute before:left-pr-16 before:top-1/2 before:inline-block before:h-pr-12 before:w-pr-1 before:-translate-y-1/2 before:bg-t-disabled before:content-[''] mo:absolute mo:-top-pr-30 mo:pl-0 mo:before:content-none";
+
+  return (
+    <CardFooter className={cardFooterStyled}>
+      {!isBestCard ? (
+        <>
+          <div className="flex items-center mo:relative mo:flex-col-reverse mo:items-start">
+            <WriterProfile writer={writer} />
+
+            <div className={createdAtStyled}>
+              <p className="text-14m text-t-disabled mo:text-12m">
+                {dotDate(createdAt)}
+              </p>
+            </div>
+          </div>
+
+          <div className="mo:flex mo:items-center mo:gap-pr-8">
+            <LikeCount type="readOnly" likeCount={likeCount} />
+
+            {userData?.id === writer.id && (
+              <div className="hidden leading-[0] mo:block">
+                <KebabDropDown
```

Comment:
만약 line height 1을 원하시는거면 따로 tailwind 속성이 있는데 이건 확인해보시고 의도하는것과 동일하면 이렇게 작성 부탁드립니다!

```suggestion
              <div className="hidden leading-none mo:block">
```

Reply:
이게 line-height를 0을 주지 않으면 불필요하게 사이즈가 잡혀버려서 0을 줘야 괜찮더라구요,,